<?xml version="1.0"?>
<launch>
  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find my_gurdy_description)/config/gurdy_control.yaml" command="load"/>

  <!-- Start only the joint state controller initially, not the position controllers -->
  <node name="initial_controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/gurdy" args="joint_state_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
    <remap from="/joint_states" to="/gurdy/joint_states" />
  </node>

</launch>